shephard-core/src/main/java/com/devsda/platform/shepherdcore/service/cacheservice/RedisCache.java

```diff
@@ -0,0 +1,77 @@
+package com.devsda.platform.shepherdcore.service.cacheservice;
+
+import com.devsda.platform.shepherdcore.service.queueservice.RabbitMQOperation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+import redis.clients.jedis.Protocol;
+
+import java.time.Duration;
+
+public class RedisCache {
+    private static final Logger log = LoggerFactory.getLogger(RedisCache.class);
+
+    final JedisPoolConfig poolConfig = buildPoolConfig();
+
+    // new JedisPool() for local redis server @6379
+    JedisPool jedisPool = new JedisPool("3.90.55.192",6379);
+    private JedisPoolConfig buildPoolConfig() {
+
+        final JedisPoolConfig poolConfig = new JedisPoolConfig();
+        poolConfig.setMaxTotal(10);
```

Comment:
What if, all 10 connections busy serving requests ? Will it create more on demand or will it hit to mysql ?